modoboa/core/app_settings.py

```diff
@@ -455,6 +455,13 @@ class GeneralParametersForm(param_forms.AdminParametersForm):
 
     sep3 = SeparatorField(label=ugettext_lazy("Miscellaneous"))
 
+    enable_inactive_accounts = YesNoField(
+        label=_("Enable inactive account tracking"),
+        initial=True,
+        help_text=_("Allow the administrator to set a threshold beyond which "
+                    "an account is considered inactive if the user hasn't logged.")
```

Comment:
We could maybe specify that threshold is in days?

Reply:
It is specified in the actual threshold field though :)